package.json

```diff
@@ -43,7 +45,6 @@
     "concurrent-transform": "^1.0.0",
     "coveralls": "*",
     "eslint": "^5.12.0",
-    "event-stream": "^4.0.1",
```

Comment:
I think "event-stream" is not needed to run the plugin itself (only for the tests). Can we leave it in the "devDependencies"?

Reply:
My error. I was having some difficulties with my local setup, npm link was doing something strange. I knocked back the version of aws-sdk to the one you have.